poi-radio/src/lib.rs

```diff
@@ -155,16 +155,22 @@ impl RadioPayloadMessage {
 
     /// Check duplicated fields: payload message has duplicated fields with GraphcastMessage, the values must be the same
     pub fn valid_outer(&self, outer: &GraphcastMessage<Self>) -> Result<&Self, BuildMessageError> {
-        if self.nonce == outer.nonce
-            && self.graph_account == outer.graph_account
-            && self.identifier == outer.identifier
-        {
+        let nonce_check = self.nonce == outer.nonce;
+        let account_check =
+            self.graph_account.to_ascii_lowercase() == outer.graph_account.to_ascii_lowercase();
```

Comment:
Could we remove the lowercase here? I think it makes more sense to require account casing to match on the two fields. tests still pass for me locally without the conversion

Reply:
optionally we can add a conversion when we use the local configured indexer address, if we make sure it is lowercased when it first get parsed into the config, then people won't have to worry about it when they supply